src/datagrids/ImportDataTable.tsx

```diff
@@ -179,17 +206,43 @@ function ImportDataTable() {
     // IMPT: requires height and width for some reason?
     <div className="mx-auto" style={{ height: '40vh' }}>
       <div className="ag-theme-alpine h-full w-full">
+        <Importheader />
+        <div className="grid-header">
+          {pageIndex * pageSize + 1}-{(pageIndex + 1) * pageSize} of {rowData.length} shown
+        </div>
         <AgGridReact
           rowData={rowData}
           columnDefs={columnDefs}
           defaultColDef={defaultColDef}
           onGridReady={onGridReady}
-          pagination
-          paginationPageSize={10}
```

Comment:
Can we use the built in pagination props and just pass the `paginationPageSize` in based on whatever is selected in the dropdown? Seems overkill to build our own pagination.

Reply:
The pagination, as it is implemented here, **does not work...**

Reply:
But can you change current pagination's style?
And I think when much data is loaded,(Of course, it's included in backend) we need to make own pagination to improve page loading speed, too.

Reply:
And you said that pagination does not work. I discussed this problem with client so I implemented only css .

Reply:
It looks like AG Grid provides some [hooks for customizing pagination](https://www.ag-grid.com/javascript-data-grid/row-pagination/#example-customising-pagination)